### DISCUSSION TITLE
Eingabefeld für Drehwinkel

### DISCUSSION THREAD

**Discussion by nicole-vlbg** [e-netze/webgis-community#81] [Ideas]
Im WebGIS4 gab es die Möglichkeit, die Ausrichtung der Karte mit dem Werkzeug "Karte rotieren" zu ändern. Dort gab es ein Feld in dem der Winkel, in dem die Karte gedreht werden sollte, eingegeben werden konnte.

Im WebGIS5 können wir unter "Karte Drucken" den Druckbereich drehen (s. Bild)

![image](https://github.com/gis-eni/webgis-forum/assets/124671123/4cd87129-10b4-4305-b98c-10197718e91e)

Kann man da auch wieder das Feld für den Drehwinkel einbauen? 
Wäre sicher Hilfreich in manchen Fällen.

**Comment by gis-eni** [comment]
Die eigentliche Karte kann man in WebGIS 5 nicht mehr drehen, weil das nativ mit Leaflet nicht möglich ist. Darum geht das, wie du schon erkannt hast nur mehr beim Drucken.

Hier geht es eher nach dem WYSIWYG Prinzip: Der Anwender passt das so an, dass alles im Ausdruck ist was er braucht. Man könnte natürlich noch ein Möglichkeit andenken, dass es hier auch eine Eingabe gibt. Ist halt ein bisserl ein Auswand und der Anwender hat wieder eine zusätzlichen Button... 

Drum die Frage: "Wäre sicher Hilfreich in manchen Fällen"? 
Wurde bisher (auch von Power Usern) bei uns noch nie angefragt. Gibt es dies "Fälle", wo das hilfreich sein könnte? ;) 
Beispiele?

**Comment by nicole-vlbg** [comment]
z.B. ein längliches Objekt in Ost-West Richtung soll zur besseren Darstellung im Ausdruck in Nord-Süd Ausrichtung also genau um 90° gedreht werden. Mit händischem Drehen ist es schwierig genau 90° zu schaffen.
Da ist es dann wichtig, dass man die Grad Werte eingeben kann.

Bei uns wurde das Eingabefeld der Gradzahl vor allem von den Waldaufsehern gerne genutzt.

**Comment by gis-eni** [comment]
Ein Eingabefeld kommt vorerst nicht. Hier gibt immer wieder Problem mit Vorzeichen, was ist die Drehrichtung etc.

Dafür wird unten rechts ein Hilfetext für die "Druck Bereich" Werkzeuge angezeigt. Dort steht dann auch immer *live* der Drehwinkel:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/58862a61-c6c7-447b-8c76-3c32da0b6ef9)

Mit dem Drehwerkzeug kann jetzt gleichzeitig die SHIFT oder STRG Taste gedrückt werden:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/7f4ebd7f-59c0-4213-bca1-add85eab0c51)

Damit "snappt" man beim drehen immer auf runde 5er oder 10er Werte. Damit ist es relativ einfach genau auf 90, 180 Grad etc zu drehen.
